### DISCUSSION TITLE
Setting Parallel values in tox.ini rather than at the command line level

### DISCUSSION THREAD

**Discussion by AidanJohnston** [tox-dev/tox#2990] [Q&A] [answered]
Hello,

Is it possible to set the parallel value inside tox.ini?  Looking through the documentation there is no mention of a parallel option -- only a CLI argument.

Something like
```
[tox]
parallel = auto
parallel_live = true
```
Of course this can be set when calling tox with `tox --parallel 0` and could be abstracted to something like a Makefile rule but I would like to be able to set it at the tox.ini level.

**Comment by gaborbernat** [comment, marked as answer]
Parallel execution is CLI-only — there is no `tox.ini` configuration for it. The intended way is via the CLI: `tox run-parallel` or `tox run --parallel`.

If you want a default, use a shell alias, a Makefile target, or a `scripts` section in your `pyproject.toml` (e.g. via `hatch` or similar).
